### PR TITLE
Add more docs disclaimers

### PR DESCRIPTION
**Pull request by jsignell**
- [x] Tests added / passed
- [x] Passes `black dask` / `flake8 dask`
- Towards #1815

### CODE REVIEW

**Comment by jsignell** [comment]
My plan is to just keep chipping away at places where `__doc__` is being explicitly set, but I'll break it up into separate PRs as suggested in https://github.com/dask/dask/pull/5709#issuecomment-565494640

File: dask/array/ufunc.py [hunk 1]
```diff
@@ -114,7 +105,8 @@ def __init__(self, ufunc):
             )
         self._ufunc = ufunc
         self.__name__ = ufunc.__name__
-        copy_docstring(self, ufunc)
+        if isinstance(ufunc, np.ufunc):
```
Comment (TomAugspurger) [inline dask/array/ufunc.py]: Can you explain why this isnstance is needed now?

Comment (jsignell) [reply dask/array/ufunc.py]: The name of the `da_frompyfunc` objects are like `frompyfunc-add` and there is no such function in numpy. The docstring from the original function is already being added to these objects in the `frompyfunc` function above.

**Comment by TomAugspurger** [comment]
SGTM. Thanks!

**Pull request merged**